Project4/style.css

```
@@ -0,0 +1,637 @@
+    filter: alpha(opacity=0);
+    opacity: 0;
+    -webkit-transform: scale(0);
+    -moz-transform: scale(0);
+    -o-transform: scale(0);
+    -ms-transform: scale(0);
+    transform: scale(0);
+    -webkit-transition: all 0.5s linear;
+    -moz-transition: all 0.5s linear;
+    -o-transition: all 0.5s linear;
+    -ms-transition: all 0.5s linear;
+    transition: all 0.5s linear;
+}
+
+.overlay-effect:hover img {
+    -webkit-transform: scale(10);
+    -moz-transform: scale(10);
+    -o-transform: scale(10);
+    -ms-transform: scale(10);
+    transform: scale(10);
+    -ms-filter: "progid: DXImageTransform.Microsoft.Alpha(Opacity=0)";
+    filter: alpha(opacity=0);
+    opacity: 0;
+}
+
+.overlay-effect:hover .mask {
+    -ms-filter: "progid: DXImageTransform.Microsoft.Alpha(Opacity=100)";
+    filter: alpha(opacity=100);
+    opacity: 1;
+}
+
+.overlay-effect:hover h3,
+.overlay-effect:hover p,
+.overlay-effect:hover a.btn {
+    -webkit-transform: scale(1);
+    -moz-transform: scale(1);
+    -o-transform: scale(1);
+    -ms-transform: scale(1);
+    transform: scale(1);
+    -ms-filter: "progid: DXImageTransform.Microsoft.Alpha(Opacity=100)";
+    filter: alpha(opacity=100);
+    opacity: 1;
+}
+
+@media (min-width: 900px) {
+
+    .overlay-item h3,
+    .overlay-item p {
+        font-size: 30px;
+    }
+}
+
+@media (min-width: 1280px) {
+    .speaker-details {
+        display: inline-block;
+        vertical-align: top;
+        width: 30%;
+    }
+
+    .overlay-item h3,
+    .overlay-item p {
+        font-size: 50px;
+
+    }
+}
+
+.demo {
+    text-align: center;
+    margin-top: 150px;
+
+}
+
+.mt {
+    margin-top: 20px;
+    font-size: 20px;
+    position: relative;
+}
+
+.medium {
+    width: 160px;
+}
+
+
+.diagonal {
+    display: block;
+    margin-right: auto;
+    margin-left: auto;
+    position: relative;
+    line-height: 50px;
+    background: #67809F;
+    color: white;
+    border: none;
+    font-weight: bold;
+    font-family: 'Open Sans',
+        sans-serif;
+    overflow: hidden;
+    z-index: 1;
+    padding: 0px;
+}
+
+.diagonal:after {
+    content: "";
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 500%;
+    height: 1000%;
+    background: #34495E;
+    z-index: -1;
+    transform-origin: 0% 0%;
+    transform: translateX(calc(20% - 25px)) translateY(10%) rotate(-45deg);
+    transform: translateY(10%) translateX(16%) rotate(-45deg);
+    transition: transform .3s;
+}
+
+.diagonal:hover::after {
+    transform: translateY(10%) translateX(-25px) rotate(-45deg);
+}
+
+.conference-image {
+    max-width: 100%;
+}
+
+header {
+    height: 100px;
+    padding: 8px 0 10px 46px;
+    background: #fff;
+    position: absolute;
+    /*change this to absolute to get that header to scroll with navbar*/
+    top: 0;
+    width: 100%;
+}
+
+header h1 {
+    font-family: 'Open Sans', sans-serif;
+    font-size: 28px;
+    color: #dddddd;
+}
+
+.main-nav,
+.main {
```

You have a `main` element, but not a "main" classname, so this isn't doing anything. To reference the element, you need to remove the period:
```suggestion
main {
```